scripts/docker_entrypoint.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# Double compilation to get correct references
+pdflatex -interaction=nonstopmode "$(basename "$1")"
+pdflatex -interaction=nonstopmode "$(basename "$1")"
+
+# Change owner of output files to the owner of the input file
+INPUT_OWNER=$(stat -c "%u:%g" "$1")
+chown "$INPUT_OWNER" "${1%.tex}.pdf" "${1%.tex}.aux" "${1%.tex}.log" "${1%.tex}.out" "${1%.tex}.pos" 2>/dev/null || true
```

Comment:
I don't like the `|| true`, it can hide errors.
I assume you did that because a file may not exist,
but you are hiding errors what can happen if you can't change the permissions.

Instead, loop over each file/extension and then check if the file exists.